### DISCUSSION TITLE
lolminer 1.38

### DISCUSSION THREAD

**Discussion by ichobmw** [Lolliedieb/lolMiner-releases#885] [General]
Hello, i am reaching out because i got 6x3060 cards first release, and so far i was getting 39.4 stable!
driver: 460.32.03 - looks like this is the most stable for me
once i updated to 1.38 - cards cant start mining..reset, reboots and so on - nothing helps
downgraded to 1.37 - so far, mining fine - 39.4 stable on ETH
i am getting few bad shares, but it its what it is...

any suggestions?

Thanks!

**Comment by rupert3k** [comment]
I'm running a FHR 3080, normally I get 98mh/s but with 1.38 it says GPU 0 detected mining lock. Unlocking & reducing down to 57mh/s.
Driver 497.09 B550 5600X Windows 11
Falling back to 1.36a for now

Thanks Lolliedieb

**Comment by michiganpinball** [comment]
I'm running 6 X RX 6600 XT on Win11.   Works fine with 1.33.  On 1.38 only 5 cards work.   Sixth card shows GAL error.

**Comment by ichobmw** [comment]
mine wont start at all...1.37 is so so stable - i am getting a lot of bad shares but i restart frequently...
mining on hiveos
